common/ingress/values.yaml

```diff
@@ -0,0 +1,11 @@
+## Configure the ingress object to hook into existing infastructure
+### ref : http://kubernetes.io/docs/user-guide/ingress/
+###
+ingress:
+  service:
+    name: example
+    port: 80
+  tls: false
+  hostname: example.com
```

Comment:
I like the host handling better the way the concourse chart does it, you can specify a list of hosts: https://github.com/kubernetes/charts/blob/5ba28e43b253928e374529066c65ab12ff458c56/stable/concourse/values.yaml#L288-L292

Though I agree that in most cases I'd use a single host.